onnxmltools/convert/sklearn/operator_converters/LeastSquares.py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._registration import register_converter
+
+
+def convert_sklearn_least_squares(scope, operator, container):
```

Comment:
Looks like you can reuse the converter for [linear regression](https://github.com/onnx/onnxmltools/blob/master/onnxmltools/convert/sklearn/operator_converters/LinearRegressor.py).